packages/language/test/linking.test.ts

```diff
@@ -318,20 +328,49 @@ describe("Linking tests", async () => {
 
       // TODO: Fix qualified name scoping
       test.skip("Must find qualified name in table", async () => {
-        await expectGotoDefinition({
+        expectGotoDefinition({
           text,
           index: 1,
           rangeIndex: 1,
         });
       });
 
       test("Must find qualified name in array", async () => {
-        await expectGotoDefinition({
+        expectGotoDefinition({
           text,
           index: 2,
           rangeIndex: 2,
         });
       });
     });
   });
+
+  test("fetch linking", async () => {
+    const doc = parse(
+      `
+        MAINPR: PROCEDURE OPTIONS(MAIN);
+        dcl A entry;
+        fetch A;
+        end MAINPR;
+    `,
+      { validate: true },
+    );
```

Comment:
Question: Should we rewrite this into a `expectGotoDefinition` test? I'm ambivalent to it.

Reply:
On second thought, just going to do this as part of the re-base here.